### DISCUSSION TITLE
Automatic Failover

### DISCUSSION THREAD

**Discussion by chirdeeptomar** [finos/TimeBase-CE#220] [Q&A]
We are looking to set up TimeBase Community on VMs with replication.

https://kb.timebase.info/community/development/howto#replication-between-two-timebase-instances

Above works only for docker, is there a documentation to set up replication for VMs and will the automatic failover happen automatically?

**Comment by alex-karpovich** [comment]
You can setup replication via TimeBase shell scripts, docker just a sample how to use it.

Please take a look here
https://kb.timebase.info/community/development/tools/shell

**Reply by chirdeeptomar** [reply]
Thanks, is there a CPU and Memory requirements for the product? Aiming to load 50TB of data across multiple streams into single instance of community edition with replication.

**Reply by alex-karpovich** [reply]
Minimal requirements:
CPU - 2 cores
Memory - 8G

Common requirements:
CPU - 4 cores
Memory - 16G

In case of high number of producers/consumers memory can be bumped to 32G
